src/i18n/en.json

```diff
@@ -84,8 +85,13 @@
       "verification-required": "Verification required"
     },
 
-    "common": {
-      "button-text-more": "More"
+    "operation-names": {
+      "3": "Tokens received",
```

Comment:
**NEVER** hardcode xdr values. They may change independently from front-end part of application